gztarchiver/doc_scraper/common/post_processing.py

```
@@ -0,0 +1,60 @@
+        save_metadata_to_filesystem(total_documents_to_process, classified_metadata_dic, config)
+        
+        # clear the temp metadata dir used by the program
+        temp_metadata_dir_path = config["output"]["metadata_dir"]
+               
+        if os.path.exists(temp_metadata_dir_path) and os.path.isdir(temp_metadata_dir_path):
+            shutil.rmtree(temp_metadata_dir_path)
+            print("Cleared the temp metadata directory")
+        else:
+            print("Temp metadata directory not exists")
+        
+    except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)